### DISCUSSION TITLE
Ability to customize order of the tabs

### DISCUSSION THREAD

**Discussion by growler420** [usebruno/bruno#1365] [Ideas]
Could there be a way to change  order of the tabs?
As of today - ver 1.6 -  the tabs are in the order:
Query
Body
Headers ,,  and so on. 

I prefer Body to be first, since I don't use Query.

**Comment by rusak47** [comment]
@helloanoop Additionally, it would be useful to be able to place request data (timeline and response) into different tab groups for the purpose of creating informative screenshots for API support. Please see the attached picture for an example.

![view_groups_config_swap_sides](https://github.com/usebruno/bruno/assets/8426197/63a85b4f-cb8d-474f-a847-2602af1aca3c)
